docs/docsite/rst/dev_guide/index.rst

```diff
@@ -1,3 +1,5 @@
+.. _developer_guide:
```

Comment:
Add links to deprecation in here

Reply:
there's a link to the deprecation page under "I want to work on other development projects" - but now that I've played with the community page more, I think this page could use some improvements

Reply:
https://github.com/ansible/ansible/pull/48734/files#diff-df5366db4400895a74aabc5e9ee4f56cR40

Reply:
Added a link in the "Working with the Ansible repo" section of the community index page as well.